<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBoxLogin">
      <property name="title">
       <string>用户登录</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayoutLogin">
       <item>
        <widget class="QLabel" name="labelUsername">
         <property name="text">
          <string>用户名：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditUsername"/>
       </item>
       <item>
        <widget class="QLabel" name="labelPassword">
         <property name="text">
          <string>密码：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditPassword">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnLogin">
         <property name="text">
          <string>登录</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxRegister">
      <property name="title">
       <string>用户注册</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayoutRegister">
       <item>
        <widget class="QLabel" name="labelRegisterUsername">
         <property name="text">
          <string>用户名：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditRegisterUsername"/>
       </item>
       <item>
        <widget class="QLabel" name="labelRegisterPassword">
         <property name="text">
          <string>密码：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditRegisterPassword">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelRegisterEmail">
         <property name="text">
          <string>邮箱：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditRegisterEmail"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnRegisterUser">
         <property name="text">
          <string>注册</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnLogout">
      <property name="text">
       <string>登出</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxUsers">
      <property name="title">
       <string>用户列表</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayoutUsers">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutSearchUser">
         <item>
          <widget class="QLineEdit" name="lineEditSearchUser">
           <property name="placeholderText">
            <string>输入用户名或邮箱关键词</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSearchUser">
           <property name="text">
            <string>搜索</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidgetUsers">
         <property name="columnCount">
          <number>1</number>
         </property>
         <property name="rowCount">
          <number>0</number>
         </property>
         <column>
          <property name="text">
           <string>用户名</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
